src/battle.ts

```diff
@@ -1868,6 +1893,7 @@ class Battle {
 				}
 				break;
 			}
+			poke.removeVolatile('furycutter' as ID);
```

Comment:
Probably just add it as a movestatus instead of having fifty `removeVolatile`s?

Reply:
The client splits `volatile`s into volatile/turnstatus/movestatus.

It'd probably be better to modify movestatus to work for this than to manually reimplement every movestatus feature.